lib/online_order.rb

```diff
@@ -0,0 +1,64 @@
+require_relative "customer"
+require "csv"
+require "pry"
+
+module Grocery
+  class OnlineOrder < Order
+    CSVFILE = "support/online_orders.csv"
+
+    attr_reader :id, :products, :customer_id
+    attr_accessor :fulfillment_status
+    def initialize(id, products, customer_id, fulfillment_status = "pending")
+      super(id, products)
+      @fulfillment_status = fulfillment_status
+      @customer_id = customer_id.to_i
+    end
+
+    #return collection of OnlineOrder instances
+    def self.all_online_order_add_arguments(id_num, hashy_hash, customer_id, fulfillment_status)
+      self.new(id_num, hashy_hash,  customer_id, fulfillment_status)
+    end
+
+    #read csv file for OnlineOrder
+    def self.get_csv_info
+      CSV.open(CSVFILE, 'r')
+    end
+
+    def self.all
+      #needs to do the same thing as Order, but change the CSV file, and get the customer id and status
+      super
+    end
```

Comment:
If a method just calls `super` with the same arguments, you can omit from the child class and Ruby will pull it from the parent instead. This applies here and to `find`.